api/StaticRange.json

```diff
@@ -0,0 +1,413 @@
+{
+  "api": {
+    "StaticRange": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/StaticRange",
+        "support": {
+          "chrome": {
+            "version_added": "60"
+          },
+          "chrome_android": {
+            "version_added": "60"
+          },
+          "edge": {
+            "version_added": null
+          },
+          "edge_mobile": {
+            "version_added": null
+          },
+          "firefox": {
+            "version_added": null
+          },
+          "firefox_android": {
+            "version_added": null
+          },
+          "ie": {
+            "version_added": null
+          },
+          "opera": {
+            "version_added": "47"
+          },
+          "opera_android": {
+            "version_added": "47"
+          },
+          "safari": {
+            "version_added": null
```

Comment:
I think it makes sense to put in the Safaris "10.1" versions in basic support. At least part of the API is supported, so I think that's sufficient

Reply:
Done :)